### ISSUE TITLE
Bug in a new project

### ISSUE THREAD

**Issue by dcuervoAG** [BGR-EGHA/LSAT#24]
Hi there, I'm trying to use the BGR/LSAT, I already installed all the libs needed for this script. However, when I try to create a new project, the python prompt cratch with this message FileNotFoundError: [Errno 2] No such file or directory: 'core\\gdal_data\\gcs.csv'. I looked for that file and it already exists.

**Comment by BGR-EGHA** [comment]
Hello dcuervoAG, 

to me it looks like your current working directory is not the LSAT directory (= wherever "startMenu_main.py" and the coresponding folders are located).
I was able to recreate your crash when I forcibly changed the working directory to "C:\":

```Traceback (most recent call last):
  File "D:\LSAT\startMenu_main.py", line 96, in newProjectLabel_clicked
    new_project = self.mainFrame.createNewProject()
  File "D:\LSAT\core\widgets\LSAT_main\MainFrame_main.py", line 479, in createNewProject
    project = NewProject()
  File "D:\LSAT\core\widgets\ProjectManager\ProjectManagement_main.py", line 48, in __init__
    with open(path) as dbfile:
FileNotFoundError: [Errno 2] No such file or directory: 'core\\gdal_data\\gcs.csv'
```

I added a line that forces the working directory to be the LSAT directory when you start LSAT, see https://github.com/BGR-EGHA/LSAT/commit/058bb2ad643e79df9c14d497aef29ae880eb449b.

I hope that fixes your problem.
If not please share more information: Which OS do you use? Which version of python? Did you follow the instructions in the readme?

Kind regards
Nick

**Comment by dcuervoAG** [comment]
Hello BGR-EGHA team and Nick,

I apologize for the delayed response, I was testing the tool. The line that you sent me works perfectly and I was able to run the script. I tried some of the methodologies such as WofE, NNA and AHP and they work with no trouble. However, with de Linear Regression method, I had some problems, so in further git-hub issues, I'll explain better what the trouble is. 

To give more context, I will share the information you requested.  My OS is windows 10 the version of python that I use is 3.7.11 and yes, I followed the readme instruction.

Finally, I want to thank you for the reply, I'm so grateful for that. 

Regards
Daniel
